__tests__/04.cross-contract-calls-with-fungible-token.spec.ts

```diff
@@ -274,19 +245,7 @@ describe(`Running on ${Runner.getNetworkFromEnv()}`, () => {
         },
         {attachedDeposit: '1', gas: '150000000000000'},
       );
-
-      const promiseOutcome = result.receipts_outcome[2].outcome;
-      if (
-        typeof promiseOutcome.status === 'object'
-        && typeof promiseOutcome.status.Failure === 'object'
-      ) {
-        // Help: update the API so hacky solutions like this aren't required
-        expect(JSON.stringify(promiseOutcome.status.Failure)).toContain(
-          'ParseIntError',
-        );
-      } else {
-        throw new TypeError('unexpected promise outcome');
-      }
+      expect(result.promiseErrorMessagesContain('ParseIntError')).toBeTruthy();
```

Comment:
Can we make this `promiseErrorMessages` instead so we can write this?

```suggestion
      expect(result.promiseErrorMessages).toContain('ParseIntError');
```

Reply:
Yeah I just had that since we wanted people to not need jest, but I do agree it reads better.